assets/css/_base.scss

```diff
@@ -19,15 +19,37 @@ h1, h2, h3, h4, h5, h6 {
 
 a {
   color: $primary-color;
+  transition: color 0.8s;
   text-decoration: none;
 
   &:hover {
     color: $lightest-color;
-    transition: color 0.8s;
   }
 }
 
+// Browsers seem to use a smaller default font-size with monospaced code
+// blocks (like 80% of the size of normal text) and that looks pretty bad with
+// small inline code-blocks in the middle of normal text (mainly because of
+// the very noticeable difference in x-height). This CSS corrects that problem.
+code {
+  font-family: monospace,monospace;
```

Comment:
```suggestion
  font-family: monospace;
```

Reply:
It's not going to work that way.

I don't exactly know the reason why (I'm not a CSS expert) but I've copy pasted this CSS from people who know it better than I do and eliminating this double-monospace redundancy was the first thing I've tried.

Here is the article I copy-pasted from: http://code.iamkate.com/html-and-css/fixing-browsers-broken-monospace-font-handling/

Another very good short (and visual) explanation of this the monospace size problem: https://www.brunildo.org/test/monospace_fsize.html

Reply:
I'm using Safari and Chrome on OSX for frontend development and testing. Both browsers need the weird double-monospace setting to correct the font size.

Reply:
I'm not a CSS expert too. :-)

It's good to know. Thanks for your explanations.